extras/agc-minimal-permissions/lib/permissions-stack.ts

```diff
@@ -13,15 +14,18 @@ export class AgcPermissionsStack extends cdk.Stack {
       description: "managed policy for amazon genomics cli admins"
     })
 
-    let agcUserPolicy = new ManagedPolicy(this, 'agc-user-policy', {
-      description: "managed policy for amazon genomics cli users"
+    let agcUserPolicyPart1 = new ManagedPolicy(this, 'agc-user-policy-part1', {
```

Comment:
Can you please change these to a reasonable name about what has been grouped as customers will see this?

Reply:
Agree, done!